packages/react-tinacms-inline/src/blocks/add-block-menu.tsx

```diff
@@ -86,24 +132,59 @@ const AddBlockButton = styled(Button)<AddMenuProps>`
   }
 
   svg {
-    height: 70%;
-    width: auto;
-    margin-right: 0.5em;
     transition: all 150ms ease-out;
   }
 
   ${props =>
-    props.open &&
+    props.isOpen &&
     css`
       svg {
         transform: rotate(45deg);
       }
     `};
 `
 
-const AddBlockWrapper = styled.div`
-  position: relative;
+interface AddBlockWrapperProps {
+  index?: number
+  offset?: number
+  position?: 'top' | 'bottom'
+  isOpen: boolean
+}
+
+const AddBlockWrapper = styled.div<AddBlockWrapperProps>(
+  p => css`
+  position: absolute;
+  left: 50%;
+  z-index: calc(var(--tina-z-index-1) - ${p.index ? p.index : 0});
```

Comment:
Should be be more specific here? 
```suggestion
  z-index: calc(var(--tina-z-index-1) - ${p.index !== undefined ? p.index : 0});
```

Reply:
or is it checking that its not 0?